### PR TITLE
Adjust container lookup pattern.

### PR DESCRIPTION
**Pull request by nathanhammond**
Minor adjustments to lookup pattern.

### CODE REVIEW

**Comment by binhums** [comment]
Looks good.

**Comment by binhums** [comment]
Happy to merge after test failures.

File: app/instance-initializers/ember-a11y.js [hunk 1]
```diff
@@ -16,12 +16,14 @@ Ember.Router.reopen({
...
-  if (arguments.length > 1) {
-    applicationInstance = arguments[1];
-  }
-  const container = applicationInstance.container ? applicationInstance.container : applicationInstance.__container__;
-  const application = container.lookup('application:main');
+// Method signatures:
+// 1.13: container, applicationInstance
+// 2.X: applicationInstance
+export function initialize(container, applicationInstance) {
+  lookup = container.lookup || applicationInstance.container.lookup;
```
Comment (Robdel12) [inline app/instance-initializers/ember-a11y.js]: Does this need `let` or `const`? Looking at the test fails it looks like this is a global

**Comment by nathanhammond** [comment]
`lookup` doesn't exist on the instance in instance-initializer in unit tests until `buildInstance` gets called, which made @BrianSipple have to dive into the `__container__`. I fixed that test, now to track down why it's not working in 1.13 anymore. 😛

**Comment by nathanhammond** [comment]
@BrianSipple Can you review this? Context:

- [Only applies to initializers, not instance-initializers.](http://emberjs.com/deprecations/v1.x/#toc_deprecate-access-to-instances-in-initializers)
- [The new default blueprint for instance-initializers.](https://github.com/ember-cli/ember-cli-legacy-blueprints/blob/master/blueprints/instance-initializer-test/qunit-files/tests/unit/instance-initializers/__name__-test.js#L10)

**Comment by BrianSipple** [comment]
@nathanhammond Good find! I tried using a using a much cleaner pattern of finding the lookup context initially, but got thrown off by the fact that the second argument would inconsistently have `.container` and `.__container__` across different tests.

**Pull request merged**